nodeadm/internal/api/net_test.go

```diff
@@ -6,21 +6,23 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestClusterDNS(t *testing.T) {
-	var tests = []struct {
+func TestGetClusterDNS(t *testing.T) {
+	tests := []struct {
 		clusterCIDR        string
 		expectedClusterDns string
 	}{
 		{
 			clusterCIDR:        "10.100.0.0/16",
 			expectedClusterDns: "10.100.0.10",
 		},
+		{
+			clusterCIDR:        "fc00::/7",
```

Comment:
is there any unhappy path we need test against? Or we already have some validation ahead to prevent the invalid case reach here?